src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -1127,6 +1128,13 @@ private void startContainer() {
     private Process getRunProcess(String command) throws IOException {
         ProcessBuilder processBuilder = new ProcessBuilder();
         processBuilder.command(getCommandTokens(command));
+        if (!System.getProperty("os.name").equalsIgnoreCase("linux")){
```

Comment:
There is an OSUtil.isWindows().  You could move this check to OSUtils as isLinux(), or just use isWindows().  Depending on whether Mac needs this case or not.

Reply:
I like this idea, we use the check many times. The same failure exists on Mac but I don't know how to create a MacOS VM.